translate/src/core/editor/actions.ts

```diff
@@ -240,7 +240,7 @@ export function endUpdateTranslation(): EndUpdateTranslationAction {
 /**
  * Save the current translation.
  */
-export function sendTranslation(
+export function sendTranslation_(
```

Comment:
With the re-export via `default`, this was proving impossible to search for as there are too many components with `sendTranslation` as a prop.

Reply:
Can we find a more meaningful name? Maybe `sendTranslationEditor`?

Reply:
I can rename it if you want, but really I'd rather work towards getting rid of the whole action, at which point it'll make sense to reconsider the name of its replacement.

Also, fair warning: This doesn't turn out to be the only case where a wrapping function has the same name as the API method. The worst is probably with `addComment`, which ends up with a second layer of wrapping, resulting in `addComment__`. I'm currently working on untangling that as a part of the fixes for #2519.